### PR TITLE
add exchange and solidity domain

### PR DESCRIPTION
**Pull request by jin-tron**

### CODE REVIEW

File: init.go [hunk 1]
```diff
@@ -217,6 +221,8 @@ func DefaultServicesConfigTestnet() Services {
 		HubDomain:          "https://hub-staging.btfs.io",
 		EscrowDomain:       "https://escrow-staging.btfs.io",
 		GuardDomain:        "https://guard-staging.btfs.io",
+		ExchangeDomain:     "https://exchange-dev.bt.co",
+		SolidityDomain:     "https://grpc.trongrid.io:50052",
```
Comment (taiyangc) [inline init.go]: can you add a migration inside `migrations.go` for people who did not have these fields (empty) to pick up? probably can check other fields to figure out testnet/mainnet

Comment (taiyangc) [inline init.go]: solidity domain is the same for testnet?? is there such a thing as a swap?

Comment (jin-tron) [reply init.go]: I saw exchange using "grpc.shasta.trongrid.io" for testnet. I will update this.

Comment (jin-tron) [reply init.go]: fixed

Comment (jin-tron) [reply init.go]: fixed

File: migrations.go [hunk 2]
```diff
@@ -103,6 +103,16 @@ func migrate_8_AnnounceDefault(cfg *Config, beforeV1B2 bool) bool {
 	return false
 }
 
+func migrate_9_WalletDomain(cfg *Config) bool {
+	if len(cfg.Services.ExchangeDomain) == 0 {
+		ds := DefaultServicesConfig()
```
Comment (taiyangc) [inline migrations.go]: Need a small check here: if the other domains are on testnet domain it should migrate to `DefaultServicesConfigTestnet()`

Comment (jin-tron) [reply migrations.go]: fixed

**Pull request merged**